src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/model/sar/SARConditionFormatter.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.hmppshdcapi.model.sar
+
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.conditions.ConditionMetadata
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.conditions.V1_CONDITIONS
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.conditions.V2_CONDITIONS
+
+typealias ConditionPolicies = Map<Int, List<ConditionMetadata>>
+
+object SARConditionFormatter {
+
+  val policyVersions = mapOf(1 to V1_CONDITIONS, 2 to V2_CONDITIONS)
+
+  // Map of condition to version (but only include conditions that only appear in a single policy version)
+  private val conditionToPolicyVersion =
+    policyVersions.entries
+      .flatMap { it.value }
+      .groupingBy { it.id }
+      .eachCount()
+      .filter { it.value == 1 }
+      .map { it.key to policyVersions.findVersion(it.key) }.toMap()
+
+  fun ConditionPolicies.getConditionText(conditionVersion: Int?, id: String): String = policyVersions[conditionVersion]?.find { it.id == id }?.text ?: id
+
+  fun ConditionPolicies.getConditionValues(values: Map<String, Any>): Map<String, Any>? = values.takeIf { it.isNotEmpty() }?.mapKeys { it.key.camelToSentenceCase() }
```

Comment:
when adding the condition variables, this makes the variable names human readable.
If the map is empty then this returns null to exclude it from the json representation